cmd/combine/main.go

```diff
@@ -0,0 +1,103 @@
+/*
+ Copyright 2021 The Tekton Authors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/empty"
+	"github.com/google/go-containerregistry/pkg/v1/mutate"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+	"github.com/google/go-containerregistry/pkg/v1/types"
+)
+
+func main() {
+	src1, src2, dst := os.Args[1], os.Args[2], os.Args[3]
+	log.Println("combining", src1, src2, "into", dst)
+	dstr, err := name.ParseReference(dst)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Add flavor so we know what's failing?

Reply:
This error message will tell us the reference it was unable to parse:

```
2021/08/03 16:46:57 could not parse reference: gcr.iofjfjfjfjf:::LKK
```